File: Tabloid/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Tabloid.Models
{
    public class Comment
    {
        public int Id { get; set; }

        public int PostId { get; set; }

        [DisplayName("Author")]
        public int UserProfileId { get; set; }

        [Required]
        public string Subject { get; set; }

        [Required]
        [DisplayName("Comment")]
        public string Content { get; set; }

        [DataType(DataType.Date)]
        public DateTime CreateDateTime { get; set; }
        public Post Post { get; set; }

        public UserProfile UserProfile { get; set; }

    }
}

File: Tabloid/client/src/components/Comment.js
import React from "react";
import { Card, CardBody } from "reactstrap";
import post from "./Post"


const Comment = ({ comment }) => {
    return (
        <Card className="m-4">

            <CardBody>
                <p>
                    <strong>{post.title}</strong>
                </p>
                <p>{comment.subject}</p>
                <p>{comment.content}</p>
                <p>{comment.displayName}</p>
                <p>{comment.createDateTime}</p>

            </CardBody>
        </Card>

    );
};
export default Comment;